UI/src/app/dashboard/core/data-factories/feature-data.js

```diff
@@ -8,99 +8,162 @@
 
 	function featureData($http) {
 		var param = '?component=';
-		
+		var agileType = {
+			kanban : "&agileType=kanban",
+			scrum : "&agileType=scrum",
+		};
+
 		var testTotal = 'http://localhost:8080/api/feature/estimates/total/Team:9933916?component=552e7b21c44a49be6261d54a';
 		var buildTotal = '/api/feature/estimates/total/';
-		
+
```

Comment:
Don't rely on local api. See this from /app/dashboard/core/data-factories/build-data.js:
        var testDetailRoute = 'test-data/build_detail.json';
        var buildDetailRoute = '/api/build/';

Put test JSON in /test-data folder.